osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -532,9 +535,9 @@ protected virtual void ApplySkin(ISkinSource skin, bool allowFallback)
         /// <param name="position">The lookup X position. Generally should be <see cref="SamplePlaybackPosition"/>.</param>
         protected double CalculateSamplePlaybackBalance(double position)
         {
-            const float balance_adjust_amount = 0.4f;
+            float balance_adjust_amount = positionalHitsoundsLevel.Value*2;
 
-            return balance_adjust_amount * (userPositionalHitSounds.Value ? position - 0.5f : 0);
+            return balance_adjust_amount * (true ? position - 0.5f : 0);
```

Comment:
Wtf?

Reply:
That was simply a placeholder whilst I wait on bdach or peppy to say it's ok to delete the old binding logic . Nothing else

Reply:
This won't pass code quality checks, and there's a missing whitespace at L538

Reply:
> This won't pass code quality checks, and there's a missing whitespace at L538

Are you at all reading what I'm saying?. I'm telling you it's a placeholder. I'll fix the whitespace.

Reply:
the `true` should just be removed, correct. suffices to use the position unconditionally.